src/Tmds.DBus/CodeGen/DynamicAssembly.cs

```diff
@@ -24,8 +24,17 @@ internal class DynamicAssembly
 
         private DynamicAssembly()
         {
-            _assemblyBuilder = AssemblyBuilder.DefineDynamicAssembly(new AssemblyName(Tmds.DBus.Connection.DynamicAssemblyName), AssemblyBuilderAccess.Run);
-            _moduleBuilder = _assemblyBuilder.DefineDynamicModule(Tmds.DBus.Connection.DynamicAssemblyName);
+            var keyStream = typeof(DynamicAssembly).GetTypeInfo().Assembly.GetManifestResourceStream("Tmds.DBus.sign.snk");
+            if (keyStream == null) throw new InvalidOperationException("'Tmds.DBus.sign.snk' not found in resources");
+            var keyBuffer = new byte[keyStream.Length];
```

Comment:
Should we hard-code these bytes in order to not have to pack `Tmds.DBus.sign.snk`?
Is this the same data as the hex-encoded `PublicKey` in `DynamicAssemblyName`?

Reply:
It is possible but looks a little strange. snk in resources add a similar amount of bytes.